pkgs/by-name/ud/udev-hid-bpf/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  bpftools,
+  clang,
+  elfutils,
+  fetchFromGitLab,
+  lib,
+  libbpf,
+  meson,
+  ninja,
+  pkg-config,
+  python3,
+  python3Packages,
+  rustPlatform,
+  systemd,
+  zlib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "udev-hid-bpf";
+  version = "2.2.0-20251121";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "libevdev";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-zZy0qjwirKYbNKFPt3SoM9993h6jOzfmg9jet/7vJ6U=";
+  };
+
+  cargoHash = "sha256-cf5FLPT581Dy4msM1hdvflN1OHij+sVdZwLHyRRYWw8=";
+
+  nativeBuildInputs = [
+    bpftools
+    clang
+    meson
+    ninja
+    pkg-config
+    python3
+  ];
+
+  buildInputs = [
+    elfutils
+    libbpf
+    systemd.dev
+    zlib
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    click
+    gitpython
+    pytest
+  ];
```

Comment:
Need a `doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;` to actually run tests.

I get a test failure in `test_sync_with_kernel_tree.py`, but it seems to be a problem with a test script itself, not the code under test. And in any case we don't use `sync_with_kernel_tree.py` so it should be safe to skip.